cmd/builder/internal/command.go

```diff
@@ -123,7 +123,7 @@ configuration is provided, ocb will generate a default Collector.
 
 func initConfig(flags *flag.FlagSet) error {
 	cfg.Logger.Info("OpenTelemetry Collector Builder",
-		zap.String("version", version), zap.String("date", date))
```

Comment:
date is never set.

Reply:
yes, it wasn't used, so it had to go. If it's in usage, let's make sure it has tests and is exercised in this repository, and then it will be maintained and we can use it in the -releases repository. This might deserve its own issue in this repository.